File: src/문자열/boj_6550.py
while True:
    try:
        s, t = input().split(" ")
        i = j = 0

        while i < len(s) and j < len(t):
            if s[i] == t[j]:
                i += 1
                j += 1
            else:
                j += 1

        print("Yes" if i == len(s) else "No")
    except:
        break

File: src/문자열/boj_6550.js
const fs = require("fs");
const stdin = fs.readFileSync("/dev/stdin").toString().split("\n");
const input = (() => {
    let line = 0;
    return () => stdin[line++];
})();

while ((ss = input())) {
    const [s, t] = ss.split(" ");
    let i = 0,
        j = 0;

    while (i < s.length && j < t.length) {
        if (s[i] === t[j]) {
            i++;
            j++;
        } else {
            j++;
        }
    }
    console.log(i === s.length ? "Yes" : "No");
}
